### PR TITLE
[#183] Add example of profiles and secrets

### PR DESCRIPTION
**Pull request by kirillmakhonin**
Documentation about secrets and profiles has been added in markdown format

### CODE REVIEW

**Comment by MaksymBilenko** [comment]
Is there secret sample coming?

File: legion/docs/source/ configuration.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+storageclass: efs              # Which storage use in PVCs
+dashboard:                     # Kubernetes dashboard configuration
+  version: "1.8.3"             # Dashboard version
+  insecure: true               # Allow insecure access
+```
+
+## Secrets 
+Each secret should be uploaded to Jenkins secrets with same name as profile (without extension).
+
+File should consist appropriate YAML formatted text.
```
Comment (alexander-semenets) [inline legion/docs/source/ configuration.md]: consists of

File: legion/docs/source/ configuration.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+### Secrets format
+Here is example of secret file
```
Comment (alexander-semenets) [inline legion/docs/source/ configuration.md]: an example

File: legion/docs/source/ configuration.md [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+fluentd:
+  aws: # Credentials for storing data on S3
+    secret_access_key: AWS_SEC_KEY
+    access_key_id: AWS_KEY_ID
+
+external_access_sgs: # list of AWS SG that should be added on ELB
+  - sg-00000000
+allowed_wan_ips:  # list of whitelisted CIDRs
+  - 20.10.30.40/32
+jenkins_cc_sg: sg-00000000 # [?]
```
Comment (alexander-semenets) [inline legion/docs/source/ configuration.md]: CC Jenkins Security group id to be whitelisted on cluster

**Pull request merged**